src/include/baselib/jni/JniEnvironment.h

```diff
@@ -136,6 +137,61 @@ namespace bl
                 return m_jniEnv -> GetVersion();
             }
 
+            jobjectRefType getObjectRefType( SAA_in const jobject object ) const NOEXCEPT
+            {
+                return m_jniEnv -> GetObjectRefType( object );
```

Comment:
No error checking needed here - GetObjectRefType returns JNIInvalidRefType if the input argument is not a valid JNI object.